packages/atlas/src/views/studio/StudioWelcomeView/StudioWelcomeView.styles.ts

```diff
@@ -1,34 +1,81 @@
+import { css } from '@emotion/react'
 import styled from '@emotion/styled'
+import { Link } from 'react-router-dom'
 
+import { SvgActionChevronR } from '@/assets/icons'
 import { SvgSigninIllustration } from '@/assets/illustrations'
-import { LimitedWidthContainer } from '@/components/LimitedWidthContainer'
+import { SvgJoystreamLogoFull } from '@/assets/logos'
+import { GridItem, LayoutGrid } from '@/components/LayoutGrid'
+import { Step } from '@/components/Step'
 import { Text } from '@/components/Text'
 import { Button } from '@/components/_buttons/Button'
-import { media, sizes } from '@/styles'
+import { cVar, media, sizes } from '@/styles'
 
-export const StyledContainer = styled(LimitedWidthContainer)`
-  margin-top: 64px;
-  display: flex;
-  flex-direction: column;
+const XXS_IMAGE_WIDTH = 360
+const XXS_IMAGE_HEIGHT = 225
+const XS_IMAGE_WIDTH = 480
+const XS_IMAGE_HEIGHT = 300
+const SM_IMAGE_WIDTH = 720
+const SM_IMAGE_HEIGHT = 450
+
+export const LEFT_ANIMATION_MD = 'left-animation-md'
+
+export const OverflowHiddenContainer = styled.div`
+  margin: 0 calc(-1 * var(--size-global-horizontal-padding)) 0;
+  overflow: hidden;
+
+  [data-aos] {
+    &[data-aos][data-aos-easing='atlas-easing'] {
+      transition-timing-function: cubic-bezier(0.09, 0.43, 0.3, 0.99);
```

Comment:
I wonder if we should add this to `GlobalStyles` since we will be using it in several places.

Reply:
yeah, I was about to do that but forgot in the end. I'll add this to `GlobalStyles`